arbitrator/include/capabilities_interface.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#ifndef __CAPABILITIES_INTERFACE_HPP__
+#define __CAPABILITIES_INTERFACE_HPP__
+
+#include <ros/ros.h>
+#include <carma_utils/CARMAUtils.h>
+#include <vector>
+#include <map>
+#include <string>
+
+namespace arbitrator
+{
+    const std::string STRATEGIC_PLAN_CAPABILITY = "strategic_plan/plan_maneuvers";
+    const std::string TACTICAL_PLAN_CAPABILITY = "tactical_plan";
+    const std::string CONTROL_CAPABILITY = "control";
```

Comment:
Does it makes sense to define those as const static in side CapabilitiesInterface class? Otherwise, it will be global to the entire arbitrator namespace.